3.process_cfret_features/scripts/1.sc_quality_control.py

```diff
@@ -103,131 +114,215 @@
 
 
 # Set the default value to 'inlier'
-plate_4_df['Outlier_Status'] = 'inlier'
+plate_4_df['Outlier_Status'] = 'Single-cell passed QC'
 
 # Update the 'Outlier_Status' column based on the outliers DataFrame using index
-plate_4_df.loc[plate_4_df.index.isin(nuclei_outliers_df.index), 'Outlier_Status'] = 'outlier'
+plate_4_df.loc[plate_4_df.index.isin(nuclei_outliers_df.index), 'Outlier_Status'] = 'Single-cell failed QC'
 
 # Create scatter plot
 plt.figure(figsize=(10, 6))
-sns.scatterplot(
+plot = sns.scatterplot(
     data=plate_4_df,
     x="Nuclei_AreaShape_Area",
     y="Nuclei_Intensity_IntegratedIntensity_Hoechst",
     hue="Outlier_Status",
+    palette={'Single-cell passed QC': '#006400', 'Single-cell failed QC': '#990090'},  # Specify colors
     alpha=0.6
 )
 
-# Add mean lines
+# Add threshold lines
 plt.axvline(
-    x=plate_4_df["Nuclei_AreaShape_Area"].mean(),
+    x=1917.0,
     color="r",
     linestyle="--",
-    label=f'Mean Area: {plate_4_df["Nuclei_AreaShape_Area"].mean():.2f}',
+    label='Min. threshold for Nuclei Area'
 )
 plt.axhline(
-    y=plate_4_df["Nuclei_Intensity_IntegratedIntensity_Hoechst"].mean(),
+    y=114.0,
     color="b",
     linestyle="--",
-    label=f'Mean Intensity: {plate_4_df["Nuclei_Intensity_IntegratedIntensity_Hoechst"].mean():.2f}',
+    label='Min. threshold for Nuclei Intensity'
 )
```

Comment:
Consider dynamically defining these thresholds

Reply:
Definitely needed since I noticed these numbers were off anyway! Thanks for catching this!